### DISCUSSION TITLE
Modelling Shrubs

### DISCUSSION THREAD

**Discussion by danci7** [UMEP-dev/UMEP#625] [Q&A]
Hello everybody, 

I'm doing a project on the effect of Vegetation on Tmrt. I want to model the impact of understorey vegetation (i.e. bushes and shrubs). My site includes trees and understorey vegetation. 
 Currently, I only have the CDSM (besides the DSM and DEM). Do I have to create a TDSM with the treeGenerator to model the Tmrt? Or which next steps would you propose?  
 
 The second question is, is it, in general, possible to model shrubs underneath trees with SOLWEIG? 
 
 Thank you very much indeed for the support and the great tool 
 Antonia

**Comment by biglimp** [comment]
Interesting project. Yes, I would recommend to create a TDSM to specify the trunk zone height of vegetation where your shrubs and bushes should have 0 meter in height. You question regrading shrubs underneath trees goes unfortunately beyond the capabilities of SOLWEIG. Since it is a 2.5D model, only one top en one bottom height of vegetation can be specified.
